regression/strings/java_char_array/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
```

Comment:
I am wondering whether these should be marked as "FUTURE" instead of "CORE"? (And then, once the features are introduced, step-by-step changed to "CORE".) This is explicitly supported by test.pl, and you can even run them as they are (using `test.pl -F`).